driver/src/main/java/org/neo4j/driver/internal/async/outbound/OutboundMessageHandler.java

```diff
@@ -77,7 +92,7 @@ protected void encode( ChannelHandlerContext ctx, Message msg, List<Object> out
 
         if ( log.isTraceEnabled() )
         {
-            log.trace( "Message %s encoded as\n%s\n", msg, prettyHexDump( messageBuf ) );
+            log.trace( "C: %s encoded as\n%s", msg, prettyHexDump( messageBuf ) );
```

Comment:
Shall we change this to "C:\n%s" so that
* it is the same as `log.trace( "S:\n%s", prettyHexDump( msg ) );`
* to avoid printing message twice as it already printed out in debug.